pkgs/os-specific/linux/cpupower/default.nix

```diff
@@ -1,22 +1,21 @@
-{ stdenv, fetchurl, kernel, coreutils, pciutils, gettext }:
+{ stdenv, buildPackages, fetchurl, kernel, coreutils, pciutils, gettext }:
 
 stdenv.mkDerivation {
   name = "cpupower-${kernel.version}";
 
   src = kernel.src;
 
-  buildInputs = [ coreutils pciutils gettext ];
+  nativeBuildInputs = [ buildPackages.gettext ];
+  buildInputs = [ coreutils pciutils ];
```

Comment:
`coreutils` doesn't seem to be needed here? Seems to build fine when dropping it, either way output has no references to it.

Reply:
It's also part of `stdenv`.